spec/unit/unmanaged_files_inspector_spec.rb

```diff
@@ -380,7 +386,7 @@ def expect_inspect_unmanaged(system, add_files, extract, filtered_paths = nil)
         "rpm", "--version"
       ).and_raise(Machinery::Errors::MissingRequirement)
 
-      expect{subject.inspect(system, description, nil)}.to raise_error(
+      expect{subject.inspect(system, description, default_filter)}.to raise_error(
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.